plugins/modules/s3_bucket.py

```diff
@@ -692,6 +763,17 @@ def get_bucket_public_access(s3_client, bucket_name):
         return {}
 
 
+def get_bucket_ownership_cntrl(s3_client, bucket_name):
+    '''
+    Get current bucket public access block
+    '''
+    try:
```

Comment:
Support for get_bucket_ownership_controls was only added to botocore in 1.18.11 (Oct 2020) this function will need to fail cleanly for older versions of botocore too.
```suggestion
    if not module.botocore_at_least('1.8.11'):
        return None
    try:
```